creds/netrc.go

```diff
@@ -63,7 +63,7 @@ func newNetrcCredentialHelper(osEnv config.Environment) *netrcCredentialHelper {
 }
 
 func (c *netrcCredentialHelper) Fill(what Creds) (Creds, error) {
-	host, err := getNetrcHostname(what["host"])
+	host, err := getNetrcHostname(what["host"][0])
```

Comment:
I noticed that in `creds/creds.go` the changes in this commit use `firstEntryForKey(what, "host")`, e.g., in `(*AskPassCredentialHelper).Fill()`.  Should the methods for `*netrcCredentialHelper` do the same (here, and in `Approve()` and `Reject()` below)?

Reply:
Ah, yes, you're right.  I'll make that change.